code/modules/security_levels/keycard authentication.dm

```diff
@@ -165,9 +165,10 @@
 			feedback_inc("alert_keycard_auth_stationRevoke",1)
 		if("Emergency Response Team")
 			if(is_ert_blocked())
-				to_chat(usr, "<span class='warning'>All Emergency Response Teams are dispatched and can not be called at this time.</span>")
+				atom_say("All Emergency Response Teams are dispatched and can not be called at this time.")
 				return
-			to_chat(usr, "<span class = 'notice'>ERT request transmitted.</span>")
+			atom_say("ERT request transmitted!")
+			command_announcer.autosay("ERT request transmitted.")
```

Comment:
Maybe include the actual reason in the message over command radio? Truncated possibly for length. That way ghosts can see why they are requesting ERT and laugh at command if they request ERT to deal with one traitor (yes, this happens).

Reply:
Good idea!